src/gluonts/mx/distribution/transformed_distribution.py

```diff
@@ -51,6 +51,14 @@ def __init__(
         self._event_shape: Optional[Tuple] = None
         self._batch_shape: Optional[Tuple] = None
 
+    @property
+    def support_min_max(self) -> Tuple[Tensor, Tensor]:
+        lb, ub = self.base_distribution.support_min_max
+        for t in self.transforms:
+            lb = t.f(lb)
+            ub = t.f(ub)
```

Comment:
Depending on whether the transformation is increasing or decreasing, one needs to invert upper and lower bound here I think. You can test this by using an affine transformation with negative scaling.

Reply:
True. This assumes increasing, or given that at this point there are only affine transformations, that the scale is positive. Not sure if we can get this info though from the transformation in general unless we know it is affine. The obvious solution is to compare lb and ub after each transformation and do something like this:
```
_lb = t.f(lb)
_ub = t.f(ub)
lb = F.minimum(_lb, _ub)
ub = F.maximum(_lb, _ub)
```
which is transformation agnostic. Any better ideas or is there any particular way you want this done?